cclib/parser/logfileparser.py

```diff
@@ -182,6 +184,7 @@ def openlogfile(filename, object=None):
         return fileobject
 
 
+@add_metaclass(ABCMeta)
```

Comment:
Can we use the standard `metaclass` syntax (https://docs.python.org/3/library/abc.html)?

Reply:
As long as we're supporting Python 2, we can't: https://stackoverflow.com/a/18513858/